src/cards/productive-time-card.ts

```diff
@@ -39,10 +57,14 @@ const getProductiveTimeData = async function (username: string, utcOffset: numbe
     // process productiveTime
     const chartData = new Array(24);
     chartData.fill(0);
+    // eslint-disable-next-line prefer-const
+    let RoundRobin = {
+        offset: 0
+    };
     for (const time of productiveTime.productiveDate) {
         const hour = new Date(time).getUTCHours(); // We use UTC+0 here
-        const afterOffset = Number(hour) + Number(utcOffset); // Add offset to hour
-        // covert afterOffset to 0-23
+        const afterOffset = adjustOffset(Number(hour) + Number(utcOffset), RoundRobin); // Add offset to hour 
```

Comment:
Could you fix this?
I am just quickly google this one.
https://stackoverflow.com/questions/53516594/why-do-i-keep-getting-delete-cr-prettier-prettier